### DISCUSSION TITLE
use really libre gpl3 license

### DISCUSSION THREAD

**Discussion by akac97** [Yet-Zio/yetCalc#203] [Polls]

**Comment by akac97** [comment]
Stronger Copyleft: GPL-3.0 ensures that any modifications or derivative works must also be released under the same license. This promotes the sharing and collaboration of improvements within the open-source community.

Protection of Openness: GPL-3.0 safeguards against the potential of someone taking open-source code, making proprietary modifications, and not contributing back to the community. This ensures that improvements are shared back with the community, enhancing the overall ecosystem.

Preventing Lock-In: GPL-3.0 prevents any single entity from turning open-source code into a proprietary product that can lock users into a specific vendor's solutions. This aligns with the philosophy of fostering a community-driven, collaborative environment.

Community Growth: GPL-3.0 encourages a collaborative culture by ensuring that contributions made to the software are shared with the community. This can lead to a more vibrant and active community, with a broader range of skills and perspectives.

Ethical Considerations: GPL-3.0 aligns with the ideology of free software and open-source movement, emphasizing the importance of providing users with the freedom to use, modify, and distribute software. This might resonate with those who value software freedom and ethics

**Comment by tuzu128** [comment]
I think this would be a nice change.
